src/pages/yonghyun/Main/Main.js

```diff
@@ -1,10 +1,157 @@
 import React from 'react';
-import { withRouter } from 'react-router-dom';
+import Nav from '../../../components/Nav/Nav';
+import Feed from './Comments/Feed';
+import './Main.scss';
 
 class Main extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      feedList: [],
+    };
+  }
+  componentDidMount() {
+    fetch('http://localhost:3000/data/totalData.json', {
```

Comment:
- [여기](https://github.com/wecode-bootcamp-korea/25-React-Westagram-3/pull/2#discussion_r719934842) 참고해주세요!

Reply:
아 그런 상황도 발생할수 있군요
수정했습니다~!!

```suggestion
    fetch('./data/totalData.json', {
```